backend/src/apiserver/template/argo_template.go

```diff
@@ -122,16 +122,21 @@ func (t *Argo) ScheduledWorkflow(modelJob *model.Job) (*scheduledworkflow.Schedu
 	// TODO: Fix the components to explicitly declare the artifacts they really output.
 	workflow.PatchTemplateOutputArtifacts()
 
-	swfParameters, err := modelToCRDParameters(modelJob.RuntimeConfig.Parameters)
+	// We assume that v1 Argo template use v1 parameters ignoring runtime config
+	swfParameters, err := stringArrayToCRDParameters(modelJob.Parameters)
 	if err != nil {
-		return nil, util.Wrap(err, "Failed to convert model parameters to CRD parameters")
+		return nil, util.Wrap(err, "Failed to convert v1 parameters to CRD parameters")
 	}
 	crdTrigger, err := modelToCRDTrigger(modelJob.Trigger)
 	if err != nil {
 		return nil, err
 	}
 
 	scheduledWorkflow := &scheduledworkflow.ScheduledWorkflow{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "argoproj.io/v1alpha1",
+			Kind:       "Workflow",
+		},
```

Comment:
Technically this isn't correct. ScheduledWorkflow itself is not Argo workflow, only part of it (spec.workflow) is.
Should we use our own kind instead, and add it to both v1 and v2 template?

Reply:
Good point. Fixed. Thank you!